### ISSUE TITLE
Can you make SmartBuff Updateable With WOWUP.IO?  

### ISSUE THREAD

**Issue by Smokenoaken** [Daeymien/SmartBuff_Unofficial#18]
It would be great if you made smartbuff updatable via WOWUP.IO's addon manager. 

Thanks! 

See Below URL

https://wowup.io/guide/section/get-addons/install-from-url

Install from URL
If you want to install an addon manually that you are unable to find using the search feature this button is for you.

Currently GitHub, WowInterface, and most CurseForge addon links work for importing/installing.

GitHub Notes
Not all GitHub repositories will work with the import process. In order for WowUp to be able to install an addon from GitHub the author must have created a tagged release. That tagged release must then contain a packaged zip file that WowUp can find.

A good example of a working GitHub repo is Teelo's Altoholic Retail.

**Comment by rcumby** [comment]
I agree.  Return Smartbuff to one of two if not both leading add-on managers.  Overwatch CurseForge and WoWUp.    Love this add-on from way back and still today.

**Comment by Daeymien** [comment]
Looking at that link, the Install from URL thingie should work as-is.

However, the addon was published to Curse as All Rights Reserved instead of an open source license.  So, I don't own the code, I have no rights to publish the code (since the author has consistently not responded to my attempts to contact them), etc.  This repository essentially exists for me putting band-aids on things for my own use and a backup in case of data loss.

**Comment by Daeymien** [comment]
Ok, looking into thigns a bit closers, I'm guessing that client needs a .zip of the release in the release for it to work so this repo won't ATM.  I'll look into it.
